lib/hawkular/alerts/alerts_api.rb

```diff
@@ -246,8 +246,18 @@ def get_alerts_for_trigger(trigger_id) # TODO: add additional filters
     # @param [Hash]criteria optional query criteria
     # @return [Array<Alert>] List of alerts in the system. Can be empty
     def list_alerts(criteria = {})
```

Comment:
May be we could deprecate `list_alerts` and `list_events`? I don't want to do that, because we would start a name-consistency war :sweat_smile: due all other methods.